tests/errors/string_01.py

```diff
@@ -2,4 +2,5 @@ def test_tuple_indicing_in_str():
     x: str = "abcdefgh"
     print(x[2, 5])
 
+
```

Comment:
```suggestion
```

Reply:
@Agent-Hellboy please apply the above change and update the reference tests (`./run_tests.py -u`).

Reply:
>Actually, I am still not checking the type of element inside the list which should be str.

Let's tackle this after the CI passes for the current changes.